Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -95,6 +95,11 @@ def polar (f : M → R) (x y : M) :=
   f (x + y) - f x - f y
 #align quadratic_form.polar QuadraticForm.polar
 
+theorem map_add_eq_polar (f : M → R) (x y : M) :
+    f (x + y) = f x + f y + polar f x y := by
+  rw [polar]
+  abel
```

Comment:
This is in #14215 with a different name

Reply:
Do you want me to rename it to `map_add`?

Reply:
Yes please; and mark it `protected`

Reply:
(or wait for #14215, which should only have one dependency left shortly...)

Reply:
I renamed for now.